packages/theme/src/styles/default-theme.css

```diff
@@ -53,16 +45,25 @@
   --swui-line-height-smaller: 14px;
   --swui-font-weight-inputs: 400;
 
+  /* Text */
+  --swui-text-primary-color: var(--lhds-color-ui-700);
```

Comment:
We use `--swui-<component-name>-...` as variables for components.
So this would conflict with the Text component. Maybe not an issue?